includes/models/model.llms.quiz.php

```diff
@@ -213,7 +226,72 @@ public function can_be_resumed_by_student( $user_id = null ) {
 			$can_be_resumed_by_student = $last_attempt && $last_attempt->can_be_resumed();
 		}
 
-		return $can_be_resumed;
+		// Check if the quiz can be resumed by a student and if the resume time limit is not reached.
+		if ( $can_be_resumed_by_student && $this->has_resume_attempt_time_expired() ) {
+			$can_be_resumed_by_student = false;
+		}
+
+		return $can_be_resumed_by_student;
+
+	}
+
+	/**
+	 * Determine if the student's resume quiz attempt time limit is expired.
+	 *
+	 * The resume quiz attempt time limit for a student is 1 day.
+	 *
+	 * @since [version]
+	 *
+	 * @return bool
+	 */
+	private function has_resume_attempt_time_expired() {
+
+		$student = llms_get_student();
+
+		if ( ! $student ) {
+			return false;
+		}
+
+		$last_attempt = $student->quizzes()->get_last_attempt( $this->get( 'id' ) );
+
+		if ( ! $last_attempt->get( 'start_date' ) ) {
+			return false;
+		}
+
+		$start_date = $last_attempt->get( 'start_date' );
+
+		/**
+		 * Filters the X time for resuming quiz.
+		 *
+		 * @since [version]
+		 *
+		 * @param int $time_period The time period in days.
+		 */
+		$time_period     = apply_filters( 'llms_quiz_resume_time_period', 1 );
```

Comment:
when moved into the attempt class this filter prefix would be `llms_quiz_attempt_`

Reply:
updated.